GDCpp/GDCpp/Runtime/InputManager.cpp

```diff
@@ -116,6 +116,18 @@ int InputManager::GetMouseWheelDelta() const {
   return mouseWheelDelta;
 }
 
+bool InputManager::IsScrollingUp() const {
```

Comment:
Almost good :) You must also declare these functions in InputManager.h (`bool IsScrollingUp() const;`). 
Declarations does not exist in JS, but are mandatory in C++. It's usually done in a .h (h for header) file.

Reply:
About this nothing to add in MouseTools.h in GDCpp ?
Something like :
```
bool GD_API IsMouseWheelScrollingUp(RuntimeScene &scene);
bool GD_API IsMouseWheelScrollingDown(RuntimeScene &scene);
```

Reply:
it doesn't seem useful to add it, the compilation works, the IDE starts, I have the conditions, and they work as expected! HOURA!
I commit this now 🐱‍🏍

The hardest part is understanding the file architecture but it's starting to come.
I still have trouble finding where to call a function when it depends on another file, there are no VSCode extensions to have a shortcut?

Reply:
Perfect! 

> it doesn't seem useful to add it

Actually it is ;)... but this will only trigger an error if missing when you use these conditions in a game compiled for the C++ game engine: the generated C++ code from events will try to call these C++ functions, but won't have any header containing them.
So for the sake of having something complete (even if we can't test it with GD5), you can add this :)

> The hardest part is understanding the file architecture but it's starting to come.

Yes, that's I think the hardest thing when browsing a codebase that was not written by you :) Let me know if you think we could write a new doc or improve an existing doc about this.

Reply:
> I still have trouble finding where to call a function when it depends on another file, there are no VSCode extensions to have a shortcut?

I'm using VS Code with the official C/C++ language extension and I have autocompletions/ctrl + click to jump to a method definition. Might need a bit of setup though to have all the proper include files in `.vscode/c_cpp_properties.json`, though I think it was already the case.

Reply:
Header file is completed in last commit 
```
bool GD_API IsMouseWheelScrollingUp(RuntimeScene &scene); 
bool GD_API IsMouseWheelScrollingDown(RuntimeScene &scene); 
```
`.vscode/c_cpp_properties.json` is good.

> Let me know if you think we could write a new doc or improve an existing doc about this.

I have few draft notes, it's fine, thank you :)